src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -1905,6 +1906,174 @@ public void requestCreationDoesNotFailWhenCirculationRulesReferenceInvalidNotice
+      hasMessage("Patron blocked from requesting"))));
+      hasParameter("reason", "Display description");
+  }
+
+  @Test
+  public void canCreateRequestWhenRequestorNotHaveActiveManualBlocks()
+    throws InterruptedException,
+    ExecutionException,
+    TimeoutException,
+    MalformedURLException {
+
+    final IndividualResource item = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource requester = usersFixture.rebecca();
+    final UUID pickupServicePointId = servicePointsFixture.cd1().getId();
+    final DateTime requestDate = new DateTime(2017, 7, 22, 10, 22, 54, DateTimeZone.UTC);
+    final DateTime expirationDate = new DateTime().plusDays(4);
+    final ManualBlockBuilder manualBlockBuilder = getManualBlockBuilder(requester, false, false, false, expirationDate);
+    final RequestBuilder requestBuilder = createRequestBuilder(item, requester, pickupServicePointId, requestDate);
+
+    loansFixture.checkOutByBarcode(item, usersFixture.jessica());
+
+    manualBlocksFixture.create(manualBlockBuilder);
+
+    Response postResponse = requestsClient.attemptCreate(requestBuilder);
+
+    assertThat(postResponse, hasStatus(HTTP_CREATED));
+  }
+
+  @Test
+  public void canCreateRequestWhenRequestorHasManualExpiredBlock()
+    throws InterruptedException,
+    ExecutionException,
+    TimeoutException,
+    MalformedURLException {
+
+    final IndividualResource item = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource requester = usersFixture.rebecca();
+    final UUID pickupServicePointId = servicePointsFixture.cd1().getId();
+    final DateTime requestDate = new DateTime(2017, 7, 22, 10, 22, 54, DateTimeZone.UTC);
+    final DateTime expirationDate = new DateTime().minusDays(1);
+    final ManualBlockBuilder manualBlockBuilder =
+      getManualBlockBuilder(requester, false, false, true, expirationDate);
+    final RequestBuilder requestBuilder = createRequestBuilder(item, requester, pickupServicePointId, requestDate);
+
+    loansFixture.checkOutByBarcode(item, usersFixture.jessica());
+    manualBlocksFixture.create(manualBlockBuilder);
+
+    Response postResponse = requestsClient.attemptCreate(requestBuilder);
+
+    assertThat(postResponse, hasStatus(HTTP_CREATED));
+  }
+
+  @Test
+  public void canCreateRequestWhenRequestorNoHaveRequestBlockAndHaveBorrowingRenewalsBlock()
+    throws InterruptedException,
+    ExecutionException,
+    TimeoutException,
+    MalformedURLException {
+
+    final IndividualResource item = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource requester = usersFixture.rebecca();
+    final UUID pickupServicePointId = servicePointsFixture.cd1().getId();
+    final DateTime requestDate = new DateTime(2017, 7, 22, 10, 22, 54, DateTimeZone.UTC);
+    final DateTime expirationDate = new DateTime().plusDays(7);
+    final ManualBlockBuilder borrowingManualBlockBuilder =
+      getManualBlockBuilder(requester, true, false, false, expirationDate);
```

Comment:
I'd prefer to construct the builder explicitly here, those booleans decreases code readability.